open-metadata-implementation/adapters/open-connectors/repository-services-connectors/search-indexing-connectors/pom.xml

```diff
@@ -0,0 +1,66 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--SPDX-License-Identifier: Apache-2.0-->
+<!--Copyright Contributors to the ODPi Egeria project.-->
+
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
```

Comment:
There's a new section we need in all poms now - look for the `<scm>` tag. This is (apparently) necessary for our release publishing process. You can cut/paste from another pom

Reply:
still need this section